ingest/Snakefile

```diff
@@ -468,11 +483,41 @@ rule prepare_files:
         """
 
 
+rule sort_fasta:
+    input:
+        sequences="results/{basename}.fasta",
+    output:
+        sorted="results/{basename}_sorted.fasta",
+    shell:
+        """
+        awk '/^>/ {{if (seq) print seq; seq=""; print; next}} {{seq = seq $0}} END {{if (seq) print seq}}' {input.sequences} | \
```

Comment:
What does this do? Some comment might be nice.
Anya just said: This turns fasta into tab separated, so each sequence and id on one line.
Can use seqkit instead here for readability.

Even better:
- Extract just the ids
- Sort the ids
- Then stream grep/join the full sequences onto the sorted ids